api/src/lib/helpers.ts

```diff
@@ -77,6 +86,58 @@ export const updateDOI = async (doi: string, publication: I.PublicationWithMetad
         if (coAuthor.user !== null) return createCreatorObject(coAuthor.user);
     });
 
+    const linkedPublications = publication?.linkedTo.map((relatedIdentifier) =>
+        relatedIdentifier.publicationToRef.type === 'PEER_REVIEW'
+            ? {
+                  relatedIdentifier: relatedIdentifier.publicationToRef.doi,
+                  relatedIdentifierType: 'DOI',
+                  relationType: 'Reviews'
+              }
+            : {
+                  relatedIdentifier: relatedIdentifier.publicationToRef.doi,
+                  relatedIdentifierType: 'DOI',
+                  relationType: 'Continues'
+              }
+    );
+
+    const doiReferences = references.map((reference) => {
+        if (reference.type !== 'DOI' || !reference.location) return;
+
+        const doi: any = getFullDOIsStrings(reference.location);
```

Comment:
eslint is warning about the use of any type here, better to use something else or allow it to infer from the return type of getFullDOIsStrings